include/hdlConvertor/createObject.h

```diff
@@ -0,0 +1,40 @@
+#pragma once
+
+#include <hdlConvertor/hdlObjects/iHdlObj.h>
+#include <hdlConvertor/hdlObjects/named.h>
+
+#include <memory>
+
+namespace antlr4
```

Comment:
I know the state of antlr4 namespace, but maybe we would like include ANTLR heather rater than self defining.

Reply:
Sure. Initial intention was to not bring in the overhead but I guess it already is somewhere in the hierarchy for every implementation file. Moreover, this header isn't intended to be used outside of the library itself. Ideally this file should be among the private headers but I didn't see any notion for it in the implementation.

I will make the change and post a commit.

Reply:
This function is used on places where ANTLR is already included because of parsers (I believe so).
If we can do it without the dynamic_cast to antlr object we would get rid of this. But I guess it is not possible.